app/src/main/java/io/gnosis/safe/ui/transactions/details/viewdata/TransactionDetailsViewData.kt

```diff
@@ -211,39 +220,62 @@ internal fun TransactionInfo.toTransactionInfoViewData(safes: List<Safe>, safeAp
     }
 
 @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-internal fun SettingsInfo?.toSettingsInfoViewData(safes: List<Safe>, safeAppInfo: SafeAppInfo? = null): SettingsInfoViewData? =
+internal fun SettingsInfo?.toSettingsInfoViewData(
+    safes: List<Safe>,
+    safeAppInfo: SafeAppInfo? = null,
+    owners: List<Owner> = emptyList()
```

Comment:
owners is not really optional here. I would reshuffle the sequence of args:
```kotlin
safes: List<Safe>,
owners: List<Owner>,
safeAppInfo: SafeAppInfo? = null
```

Reply:
Done